src/16F740.c

```diff
@@ -291,7 +291,7 @@ void btl_state_update_victory(void) {
             player->transStatus = 0;
             player->koStatus = 0;
             player->koDuration = 0;
-            player->debuffIcon->ptrPropertyList[15] = 0;
+            ((s32*)player->debuffIcon->data)[15] = 0;
```

Comment:
This typing is looking weird; do we have `data` typed wrong or something? cc @ethteck

Reply:
we ideally should cast to the appropriate effect data struct, but we may not have one for this effect? for now I think it's ok but maybe should be left with a todo (or even remove the s32* cast to keep a warning there as a reminder to us)

Reply:
`/* 0x0C */ void* data; // often but not always EffectInstanceData`
Since I don't know anything about effects, I decided to leave it without casting to EffectInstanceData

Reply:
yeah that's fair. we can address this later